gtsam/hybrid/HybridFactorGraph.cpp

```diff
@@ -70,8 +70,8 @@ HybridFactorGraph HybridFactorGraph::linearize(
 
 bool HybridFactorGraph::equals(const HybridFactorGraph& other,
                                double tol) const {
-  return Base::equals(other, tol) &&
-         nonlinearGraph_.equals(other.nonlinearGraph_, tol) &&
+  // Base::equals(other, tol) && TODO(fan): this does not work because Factor::equals should be virtual!!!
```

Comment:
Then we should make Factor::equals virtual. :)

Reply:
Yes, but I don't want to touch it now, thus this comment :)

Reply:
Sounds good!